lib/publish_static_pages.rb

```diff
@@ -1,42 +1,63 @@
 module PublishStaticPages
   PAGES = [
+    {
+      content_id: "dbe329f1-359c-43f7-8944-580d4742aa91",
```

Comment:
I think I wrote this on Slack last week, but I'll write it here so it doesn't disappear into the ether.

The publishing of this page would probably be more suited to remain in Whitehall, as that is a publishing app and that's where the patched links will be coming from.

The change that will need to be made in Whitehall is to update the `rendering_app` to be `government-frontend`, then ensure the links get populated when the special routes task is published.

That would mean all the changes to `PublishStaticPages` here won't be needed, which should make things simpler.

Reply:
https://github.com/alphagov/whitehall/pull/6303